### ISSUE TITLE
pcmanfm-qt: bookmark name not shown correctly.

### ISSUE THREAD

**Issue by ArthurBorsboom** [lxde/lxqt#278]
I have a couple of bookmarks in pcmanfm (extracted from .gtk-bookmarks):

```
file:///home/arthur/Downloads Downloads
file:///home/arthur/Documents/Work/xxx/Projects Projects
file:///home/arthur/Documents/Work/xxx/Technical Technical
file:///home/archive archive
sftp://arthur@xxx.xxx.com:2222/u01/oracle/PrimaveraGateway PrimaveraGateway
smb://server01.xxx.com/applic applic
```

All the bookmarks show the correct name, except the SMB one, which shows as a '/' in pcmanfm. Changing the name, does not work / does not help. Besides it seems to be stored in the file above.

![2014-09-12-114611_817x587_scrot](https://cloud.githubusercontent.com/assets/6639859/4248005/a5595804-3a61-11e4-8cbb-61e41dd805fc.png)

**Comment by PCMan** [comment]
@dfrizel I believed that this issue has been fixed in pcmanfm-qt.
Now we only save gtk 3.0 bookmarks file.
Since gtk 2.24.12, gtk2 reads the gtk3 bookmarks file as well so this works.
I'm closing the issue for now as it's already fixed.
Would you please try the latest pcmanfm-qt again in git?
If it does not work for you, please re-open the issue.
Thanks!

**Comment by ArthurBorsboom** [comment]
Confirmed this is fixed in v0.9
